runtime/common/src/tests/orml_xcm.rs

```diff
@@ -0,0 +1,266 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>
+
+use frame_support::{
+	dispatch::RawOrigin,
+	traits::{EnsureOrigin, OriginTrait},
+	*,
+};
+
+use sp_runtime::traits::Dispatchable;
+
+use crate::{
+	tests::setup::{bob, ExtBuilder},
+	BaseRuntimeRequirements, EnsureRootOrTwoThirdsCouncil, FilterEnsureOrigin,
+};
+use primitives::*;
+use xcm::latest::prelude::*;
+use xcm_executor::traits::Convert;
+
+pub fn orml_xcm_root_works<
+	Origin: OriginTrait
+		+ From<RawOrigin<AccountId>>
+		+ Clone
+		+ std::convert::From<
```

Comment:
This restriction is suspicious. Please check and feedback the reason for doing so.